v2/test/mysql_test.go

```diff
@@ -417,7 +417,7 @@ func newSecret(tc *testcommon.KubePerTestContext, key string, password string) *
 func newMySQLServer(tc *testcommon.KubePerTestContext, rg *resources.ResourceGroup, adminUsername string, adminKey string, adminSecretName string) *mysql.FlexibleServer {
 	// Force this test to run in a region that is not capacity constrained.
 	// location := tc.AzureRegion TODO: Uncomment this line when West US 2 is no longer constrained
-	location := to.StringPtr("East US")
+	location := to.StringPtr("Australia East")
```

Comment:
I'm surprised using the display name for the region works - would have expected to need `australiaeast` or `eastus`.

Suggest changing to use the region code as that's more standard.